installer/cli/.env

```diff
@@ -35,3 +35,8 @@ COMPOSE_PROJECT_NAME=streampipes
 # 
 # default: activemq
 #SP_MQTT_HOST=mosquitto
+
+
+# For database migration in v0.91.0 - set init mode to 'upgrade' to migrate an existing installation
+SP_INFLUX_INIT_MODE=setup
+#SP_INFLUX_INIT_MODE=upgrade
```

Comment:
We should provide an upgrade guide somewhere and also shortly describe this change in the PR's description (for the release notes)

Reply:
I created a ticket for this: #1206